nodewatcher/modules/frontend/ip_space/static/js/jquery.tipsy.js

```diff
@@ -0,0 +1,284 @@
+//Author: https://github.com/jaz303
```

Comment:
Where is this coming from? What are changes? Why not simply use  git submodule and pull that code in? Or make a fork of it and have changes there and then git submodule here?